frontend/src/components/ScoreCard.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+
+import { PopulatedReview } from "../api";
+import { formatFieldNameHumanReadable } from "../helpers/review";
+
+interface ScoreCardProps {
+  review: PopulatedReview;
+}
+
+export default function ScoreCard({ review }: ScoreCardProps) {
+  // Each field in this array is [fieldName, fieldValue, maxValue, rubricLink]
+  const scoreAndRatingFields = Object.entries(review.stage.fields || {})
+    .filter(([key, _]) => key.includes("score") || key.includes("rating"))
+    .map(([key, _]) => [
+      key,
+      review.fields[key],
+      review.stage.fields[key].maxValue,
+      review.stage.fields[key].rubricLink,
+    ]);
+
+  return (
+    <div className="tw:flex tw:flex-col tw:rounded-lg tw:overflow-hidden tw:min-w-90">
+      <div className="tw:bg-accent tw:p-2.5 tw:text-white tw:flex tw:justify-between tw:items-center">
+        <span>{review.reviewerEmail ?? "(Unassigned)"}</span>
```

Comment:
Here and in `StageNotes`: can we show the reviewers' names instead of just emails? you can use the `userUsers` hook I created, then look up name by email similar to [here](https://github.com/TritonSE/TSE-Fulcrum/blob/main/frontend/src/views/ReviewsView.tsx#L142)